### PR TITLE
Rename --files to --contents, improve local default URLs

### LINKED ISSUE #277: Notebooks are not copied over in build
Hi! This seems like a cool project. I'd like to use it for teaching a course.

## Description

Setting up a simple site with a simple prepared notebook does not show the notebook in lab/

## Background

I want to prepare a static web site based on a few python kernel notebooks, and with some python extensions pre-installed.
I find the documentation on how to get started a bit confusing; the examples/ directory does not explain how to build it and which configuration files are important. Here is what I did:

## Reproduce

1. Create and enter a empty directory
2. Create a jupyter notebook files/Untitled.ipynb with one cell: "import matplotlib"
3. run: jupyter lite init
4. run: jupyter lite build
5. run: jupyter lite serve
6. open http://127.0.0.1:8000/lab
7. be told "HTTP 403: Forbidden (lab is not a file)"
8. open http://127.0.0.1:8000/lab/ instead
9. file list (left panel) is empty
10. Go to the user doc at https://jupyterlite.readthedocs.io/en/latest/ -- it is pretty empty.
11. Go to the developer doc at https://jupyterlite.readthedocs.io/en/latest/configuring.html. 
   * It says "jupyter lite build" will copy all files in files/, but it did not (`find _output/ -iname '*.ipynb'` is empty)

## Expected behavior

1. run: jupyter lite init
2. create notebook files in files/
3. run: jupyter lite build
4. run: jupyter lite serve
5. on the quoted url, the notebooks are shown.

It would be helpful if the documentation answered the following questions:
 * How do I include a python packages so it comes with the static site?
 * How do I write the jupyter-lite.json, jupyter-lite.ipynb, index.html files? (beyond the schema and long unreadable example blurb)
 * How is the examples/ folder in the github repo built & delivered? Which config files are important?

## Context

- Operating System and version: Ubuntu Linux
- Browser and version: Firefox
- JupyterLite version: 0.1.0a5

### PR DESCRIPTION
**Pull request by bollwyvl**
## References

- fixes #277

## Code changes

- [x] renames `--files` to `--contents`
  - they're still _served_ from `/files`, but makes it a bit more clear (so _many_ files, but not so much _content_) 
- [x] hopefully has better "doorknob" feels for `serve` by better handling (a lack of) `index.html`

## User-facing changes

- improves some docs, better explain contents merging

## Backwards-incompatible changes

- CLI changes, noted above...

### CODE REVIEW

File: app/config-utils.js [hunk 1]
```diff
@@ -233,7 +233,7 @@ function addFavicon(config) {
 async function main() {
   const config = await jupyterConfigData();
   if (config.baseUrl === new URL(here()).pathname) {
-    window.location.href = config.appUrl;
+    window.location.href = config.appUrl.replace(/\/?$/, '/index.html');
```
Comment (bollwyvl) [inline app/config-utils.js:236]: This is the place that was causing the various servers to get confused...

**Comment by bollwyvl** [comment]
Ick:

```
.  post_build:contents:contents:.cache
TaskError - taskid:post_build:contents:contents:.cache
PythonAction Error
```

**Comment by bollwyvl** [comment]
Gah, lost a point of coverage, but otherwise this is ready for review.

File: examples/retro/jupyter-lite.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "jupyter-lite-schema-version": 0,
+  "jupyter-config-data": {
+    "disabledExtensions": ["@jupyterlab/drawio-extension"]
```
Comment (bollwyvl) [inline examples/retro/jupyter-lite.json:4]: for #286

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**